docker-compose/common-services/anchor-platform-config.yaml

```diff
@@ -0,0 +1,335 @@
+version: 1
+
+host_url: http://host.docker.internal:8080
+
+stellar_network:
+  network: TESTNET
+  network_passphrase: Test SDF Network ; September 2015
+  horizon_url: https://horizon-testnet.stellar.org
+
+callback_api:
+  base_url: http://host.docker.internal:8081
+  auth:
+    type: NONE
+
+platform_api:
+  auth:
+    type: NONE
+  base_url: http://host.docker.internal:8080
+
+languages: en
+
+logging:
+  level: WARN
+  stellar_level: INFO
+
+#data:
+#  type: sqlite
+#  database: anchor-platform
+#  initial_connection_pool_size: 2
+#  max_active_connections: 10
+#  flyway_enabled: true
+#  ddl_auto: update
+
+data:
+  type: postgres
+  server: localhost:5440
+  database: postgres
+#  username: "postgres"
+#  password: "password"
+  initial_connection_pool_size: 3
+  max_active_connections: 10
+  flyway_enabled: true
+  ddl_auto: update
+
+sep1:
+  enabled: true
+  toml:
+    type: string
+    value: |
+      ACCOUNTS = []
+      VERSION = "0.1.0"
+      NETWORK_PASSPHRASE = "Test SDF Network ; September 2015"
+      SIGNING_KEY = "GCHLHDBOKG2JWMJQBTLSL5XG6NO7ESXI2TAQKZXCXWXB5WI2X6W233PR"
+      
+      WEB_AUTH_ENDPOINT = "http://localhost:8080/auth"
+      KYC_SERVER = "http://localhost:8080/sep12"
+      TRANSFER_SERVER_SEP0024 = "http://localhost:8080/sep24"
+      DIRECT_PAYMENT_SERVER = "http://localhost:8080/sep31"
+      ANCHOR_QUOTE_SERVER = "http://localhost:8080/sep38"
+      
+      [[CURRENCIES]]
+      code = "USDC"
+      issuer = "GDQOE23CFSUMSVQK4Y5JHPPYK73VYCNHZHA7ENKCV37P6SUEO6XQBKPP"
+      status = "test"
+      is_asset_anchored = true
+      anchor_asset_type = "fiat"
+      desc = "A test USDC issued by Stellar."
+      
+      [[CURRENCIES]]
+      code = "JPYC"
+      issuer = "GDQOE23CFSUMSVQK4Y5JHPPYK73VYCNHZHA7ENKCV37P6SUEO6XQBKPP"
+      status = "test"
+      is_asset_anchored = true
+      anchor_asset_type = "fiat"
+      desc = "A test JPYC issued by Stellar."
+
+sep10:
+  enabled: true
+  home_domain: localhost:8080
+
+sep12:
+  enabled: true
+
+sep24:
+  enabled: true
+  interactive_url:
+    base_url: http://host.docker.internal:8091/sep24/interactive
+
+sep31:
+  enabled: true
+  payment_type: STRICT_SEND
+  deposit_info_generator_type: self
+
+sep38:
+  enabled: true
+
+metrics:
+  enabled: true
+  extras_enabled: true
+  run_interval: 30
+
+events:
+  enabled: true
+  event_type_to_queue:
+    quote_created: ap_event_quote_created
+    transaction_created: ap_event_transaction_created
+    transaction_status_changed: ap_event_transaction_status_changed
+    transaction_error: ap_event_transaction_error
+  publisher:
+    type: kafka
+    kafka:
+      bootstrap_server: host.docker.internal:29092
+      client_id:
+      retries: 1
+      linger_ms: 1000
+      batch_size: 10
+
+assets:
+  type: json
```

Comment:
The assets can be splitted to another file.

Reply:
Splitted